meta-coav/recipes-coav/coav-control/coav-control_0.0.1.bb

```diff
@@ -0,0 +1,37 @@
+# coav-control.bb
+SUMMARY = "Collision avoidance test application."
+HOMEPAGE = "https://github.com/01org/collision-avoidance-library"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=93888867ace35ffec2c845ea90b2e16b"
+
+DEPENDS = "glm librealsense python-future"
+RDEPENDS_${PN} = "librealsense"
+
+SRCREV = "${AUTOREV}"
+SRC_URI = "gitsm://git@github.com/01org/collision-avoidance-library.git;protocol=https;branch=master"
```

Comment:
It does. it looks like gibberish of "git protocol". It was tested using my user at dronesonia that doesn't have that.